code/game/machinery/vending.dm

```diff
@@ -42,7 +42,12 @@
 	layer = 2.9
 	anchored = 1
 	density = 1
+	pass_flags = null

```

Comment:
Careful. If this uses bitflags this may cause issues with bump stuff. I don't have time to check right now how it works but it's worth making sure.